docs/en/sql-reference/functions/tuple-map-functions.md

```diff
@@ -220,6 +220,8 @@ Result:
 
 Returns all keys from the `map` parameter.
 
+Can be optimized by setting the [optimize_functions_to_subcolumns](../../operations/settings/settings.md#optimize-functions-to-subcolumns). With `optimize_functions_to_subcolumns = 1` the function reads subcolumn [keys](../../sql-reference/data-types/map.md#subcolumn-keys).
```

Comment:
```suggestion
Can be optimized by enabling the [optimize_functions_to_subcolumns](../../operations/settings/settings.md#optimize-functions-to-subcolumns) setting. With `optimize_functions_to_subcolumns = 1` the function reads only [keys](../../sql-reference/data-types/map.md#subcolumn-keys) subcolumn instead of reading and processing the whole column data.
```